test/games_test.rb

```diff
@@ -79,19 +89,23 @@ def test_it_can_determine_venue_with_fewest_games
   def test_it_can_total_goals_by_season
     hash = {:"20122013" => 15, :"20162017" => 9, :"20172018" => 11}
 
-    assert_equal hash, @games.total_goals_season(:season)
+    assert_equal hash, @games.total_goals_stat(:season)
   end
 
   def test_it_can_average_goals_by_season
     hash = {:"20122013" => 5, :"20162017" => 9, :"20172018" => 5.5}
 
-    assert_equal hash, @games.average_goals_season(:season)
+    assert_equal hash, @games.average_goals_stat(:season)
   end
 
   def test_it_can_average_goals_per_game
     assert_equal 5.83, @games.average_goals_game
   end
 
+  def test_it_can_count_all_games
+    assert_equal 6, @games.stats.count
```

Comment:
Unnecessary test really. But we found (and used in a couple methods) that just @stats.count was a real easy way to get the count of total games played.